schema/primitives/objects/transactions/consolidation/Consolidation.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/primitives/objects/transactions/consolidation/Consolidation.schema.json",
+  "title": "Primitive - Security Consolidation Transaction",
+  "description": "Abstract object describing common properties to a consolidation of a security",
+  "type": "object",
+  "properties": {
+    "security_ids": {
+      "title": "Security Consolidation - Incoming Securities ID Array",
+      "description": "Identifier for the securities (stock, plan security, warrant, or convertible) that are being consolidated into one position by which it can be referenced by other transaction objects. Note that while this identifier is created with an issuance object, it should be different than the issuance object's `id` field which identifies the issuance transaction object itself. All future transactions on the security (e.g. acceptance, transfer, cancel, etc.) must reference this `security_id` to qualify which security the transaction applies to.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "resulting_security_id": {
```

Comment:
@arthur-clara, can you use a minItems validation here to ensure there are two or more ids? See here for an example: https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/7abfda1d5332e816c09b387db95a5f57afbafbb7/schema/primitives/objects/transactions/transfer/Transfer.schema.json#L16-L24

Reply:
Actually, thinking about #471, @arthur-clara, maybe we can make this more general purpose? Like a multiple security transaction primitive which you can compose into your consolidation transaction. See more here - https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/pull/471#issuecomment-1967283436

Reply:
This should be "MultipleSecurityTransaction" or something similar.